src/mame/sega/suprloco.cpp

```diff
@@ -328,26 +421,49 @@ void suprloco_state::draw_sprites(bitmap_ind16 &bitmap, const rectangle &cliprec
 
 uint32_t suprloco_state::screen_update(screen_device &screen, bitmap_ind16 &bitmap, const rectangle &cliprect)
 {
-	m_bg_tilemap->draw(screen, bitmap, cliprect, 0, 0);
+	if (!BIT(m_control,4)) // force blank
+	{
```

Comment:
Still missing a space after comma on this one.

Reply:
This is now fixed.